.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: node_js
+node_js:
+  - "6"
+  - "8"
+  - "9"
+  - "10"
+  - "11"
+
```

Comment:
Could we also add the following so it tests on the different OSs too:
```
os:
  - windows
  - linux
  - osx
```
detailed [here](https://blog.travis-ci.com/2018-10-11-windows-early-release)

Reply:
I've now added these